doc/sphinx/changes.rst

```diff
@@ -2,6 +2,441 @@
 Recent changes
 --------------
 
+Version 8.10
+------------
+
+Summary of changes
+~~~~~~~~~~~~~~~~~~
+
+|Coq| version 8.10 contains two major new features: support for a native
+fixed-precision integer type and a new sort `SProp` of strict
+propositions. It is also the result of refinements and stabilization of
+features, deprecations or removals of deprecated features, cleanups of
+the internals of the system and API. This release includes many
+user-visible changes, including deprecations that are documented in
+``CHANGES.md`` and the following section, and new features that are
+documented in the reference manual. Here are the most important
+user-visible changes:
+
+- Kernel:
+
+  - A notion of primitive was added, whose first instance is
+    primitive cyclic unsigned integers, axiomatized in module
+    UInt63(link), by Maxime Dénès, Benjamin Grégoire and Vincent
+    Laporte.
+
+  - The `SProp` sort of definitionally proof-irrelevant propositions was
+    introduced by Gaëtan Gilbert. `SProp` allows to mark proof terms as
+    irrelevant for conversion, and is treated like `Prop` during
+    extraction. It is enabled using `-allow-sprop`.
+
+  - The unfolding heuristic in termination checking was made more complete
+    by Enrico Tassi, allowing more constants to be unfolded to discover
+    valid recursive calls.
+
+- Notations:
+
+  - New command `String Notation` to register string syntax for custom
+    inductive types, by Jason Gross.
+
+  - Numeral Notations now parse decimal constants, by Pierre Roux.
+
+- Ltac:
+
+  - Ltac backtraces can be turned on using the "Ltac Backtrace" option,
+    by Pierre-Marie Pédrot.
+
+- Universes:
+
+  - Added `Print Universes Subgraph` variant of `Print Universes` and
+    private universes for opaque polymorphic constants, by Gaëtan Gilbert.
+
+- The tactics 'lia','nia','lra','nra' are now using a novel
+  Simplex-based proof engine, by Fréderic Besson.
+
+- `Combined Scheme` now works when inductive schemes are generated in sort
+  `Type`, by Théo Winterhalter.
+
+- CoqIDE:
+
+  - Migrated to gtk+3 and lablgtk3 by Hugo Herbelin.
+
+  - Supports smart input for Unicode characters by Arthur Charguéraud.
+
+- SSReflect has new intro patterns and a consistent clear discipline, by
+  Enrico Tassi.
+
+- Build system:
+
+  - The project is migrating to `dune`, by Emilio Jesús Gallego Arias.
```

Comment:
Thanks @Zimmi48 , I'd write "which is scheduled for deprecation" instead of the removal. Also, worth adding could be "the Dune build system provides an improved experience for plugin developers".

Reply:
We should also credit @rgrinberg here and thank the Dune team.